packages/itwin/property-grid/src/components/SettingsDropdownMenu.tsx

```diff
@@ -112,7 +112,7 @@ export function SettingsDropdownMenu({ settingsMenuItems, dataProvider }: Settin
   const menuItems = (close: () => void) => settingsMenuItems.map((item, index) => <Fragment key={index}>{item({ dataProvider, close })}</Fragment>);
 
   return <DropdownMenu menuItems={menuItems}>
-    <IconButton styleType="borderless" size="small" title={PropertyGridManager.translate("settings.label")}>
+    <IconButton styleType="borderless" title={PropertyGridManager.translate("settings.label")}>
```

Comment:
Have you tried making search bar small? If it looks bad with small search bar I think there are more buttons where you need to remove `size: "small"` (ancestor navigation, more elements button)